### PR TITLE
code to print fibonacci sequence upto n terms

### PR DESCRIPTION
**Pull request by aniketrathi**
## Description
Onboarding Exercise 1 for day 1, print fibonacci sequence upto n terms

## Database schema changes
None

### Automated test cases added
- None

### Manual test cases run
- None

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+let n1 = 0, n2 = 1, nextTerm;
+
+console.log('Fibonacci Series:');
+
+for (let i = 1; i <= number; i++) {
+    console.log(n1);
+    nextTerm = n1 + n2;
+    n1 = n2;
+    n2 = nextTerm;
+}
```
Comment (GaurisankarJ) [inline index.js]: Please add a new line at the end.

**Comment by GaurisankarJ** [comment]
Use `.gitignore` to remove `node_modules` from git.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+const prompt = require("prompt-sync")({ sigint: true });
+// take input from the user
+const number = parseInt(prompt("Enter the number of terms: "));
+let n1 = 0, n2 = 1, nextTerm;
+
+console.log('Fibonacci Series:');
+
+for (let i = 1; i <= number; i++) {
+    console.log(n1);
```
Comment (jjalan) [inline index.js]: 2 spaces instead of 4

File: .gitignore [hunk 3]
```diff
@@ -0,0 +1 @@
+node_modules
```
Comment (jjalan) [inline .gitignore]: add new line

**Review by jjalan — CHANGES_REQUESTED** [review]
do not checkin node modules.

**Review by GaurisankarJ — APPROVED** [review]

**Pull request merged**